header.php

```diff
@@ -0,0 +1,15 @@
+<div class="navbar">
+            <img src="CSS/Images/logo.png" alt="" class="logo">
+            <h2>ADMIN</h2>
+            <nav>
+              <input type="text" placeholder="Search" title="Type in a category" id="search">
+              <ul>
+                  <li><a href="index.php" target="_blank">Dashboard</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="manageTickets.php" target="_blank">Manage Ticket</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="normalTickets.php" target="_blank">Normal Ticket</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="foreignerTickets.php">Foreigner Ticket</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="Report.php">Reports</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
```

Comment:
- [ ] Please remove all the target attributes from the anchor element so that your nav links will not open a new tag on your browser.